src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts

```diff
@@ -267,34 +260,35 @@ describe('Exercise Scores Component', () => {
         participation.team = team;
         const expectedResult = 'name (shortName) ⟹ name1 (login1), name2 (login2)';
 
-        expect(component.searchResultFormatter(result)).to.equal(expectedResult);
+        expect(component.searchResultFormatter(result)).toBe(expectedResult);
 
         participation.team = undefined;
     });
 
     it('should search result and return empty string', () => {
-        expect(component.searchResultFormatter(result)).to.equal('');
+        expect(component.searchResultFormatter(result)).toBe('');
     });
 
     it('should search text from result', () => {
-        expect(component.searchTextFromResult(result)).to.equal('participationId');
+        expect(component.searchTextFromResult(result)).toBe('participationId');
     });
 
     it('should refresh properly', () => {
-        const resultServiceStub = sinon.stub(resultService, 'getResults').returns(of(new HttpResponse<Result[]>({ body: [result] })));
+        const resultServiceStub = jest.spyOn(resultService, 'getResults').mockReturnValue(of(new HttpResponse<Result[]>({ body: [result] })));
 
         component.refresh();
 
-        expect(resultServiceStub).to.have.been.calledOnceWith(component.exercise);
-        expect(component.results).to.have.same.members([result]);
-        expect(component.isLoading).to.be.false;
+        expect(resultServiceStub).toHaveBeenCalledTimes(1);
+        expect(resultServiceStub).toHaveBeenCalledWith(component.exercise);
+        expect(component.results).toContainValues([result]);
```

Comment:
```suggestion
        expect(component.results).toEqual([result]);
```
Using the basic Jest function should yield the same result here.